Microsoft.Toolkit.Uwp.UI.Behaviors/InfoBar/NotificationWithOverrides.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Microsoft.Toolkit.Uwp.UI.Behaviors
+{
+    /// <summary>
+    /// An extended notification request.
+    /// It allows the user to override different parts of the targeted <see cref="Microsoft.UI.Xaml.Controls.InfoBar"/>.
+    /// Only the explicitly set properties will be applied to the InfoBar.
+    /// </summary>
+    public class NotificationWithOverrides : Notification
```

Comment:
For the sake of simplicity, could the properties in this -Overrides class, just exist in the parent Notification class?  And then we'd only have the one class?

Reply:
At first I had a single class and though that having a regular class and one more tailored to "advanced" scenario could be useful but you're right, I the end, it makes the API surface more complicated.
I will merge the two